protoc-gen-gogo/generator/generator.go

```diff
@@ -715,8 +715,13 @@ var pkgNamesInUse = make(map[string][]*FileDescriptor)
 // Pkg is the candidate name.  If f is nil, it's a builtin package like "proto" and
 // has no file descriptor.
 func RegisterUniquePackageName(pkg string, f *FileDescriptor) string {
-	// Convert dots to underscores before finding a unique alias.
-	pkg = strings.Map(badToUnderscore, pkg)
+	// Only make pkg names unique if they are non-standard lib
+	if f != nil {
```

Comment:
Nice trick here :)

Reply:
This comment is still wrong (this code only makes the names unique if they are *not* builtin).

However, this makes me think that this comment is just not useful. Perhaps instead of explaining what this code is doing (which is pretty clear just from reading the code and the method comment), explain **why** it is doing it?

Reply:
How about this:
```go
	// Ensure that package names across multiple imports are unique, unless they
	// are builtin or standard lib packages, in which case they should not conflict.
```